routes/web.php

```diff
@@ -392,6 +392,8 @@
     Route::post('/events/{event}/by/notification', [EventController::class, 'destroyByNotification'])
         ->name('events.delete.by.notification');
     Route::delete('/events/{event}/shifts', [EventController::class, 'destroyShifts'])->name('events.shifts.delete');
+
+    //Route is doubly used. So only the second line (decline) applies
     Route::put('/event/requests/{event}', [EventController::class, 'acceptEvent'])->name('events.accept');
```

Comment:
Both routes map to `/event/requests/{event}`  therefore only the latter (decline route) is actually callable. 
Are these obsolete, or is it a bug?